keyboards/handwired/split_cloud/readme.md

```diff
@@ -0,0 +1,16 @@
+# Split-Cloud handwired keyboard
+
+The Split-Cloud is a split ortholinear 5x12 layout, forked from the Preonic and Nyquist keyboard models. The default layout adds extra options for some added game compatibility, two control keys for QEmu, and adds a steno layer based on the Planck example.
+
+![Split-Cloud](https://raw.githubusercontent.com/coolsa/cloud-keyboard/main/images/finalProduct.jpg)
+
+* Keyboard Maintainer: [Cloud Chagnon](https://github.com/coolsa)
+* Hardware Supported: Pro Micro
+* Hardware Availability: [Repository](https://github.com/coolsa/split-cloud)
+
+Make example for this keyboard:
+
+    make handwired/split_cloud:default:avrdude-split-left
+    make handwired/split_cloud:default:avrdude-split-right
```

Comment:
```suggestion
    make handwired/split_cloud:default
```

Reply:
I altered the readme to have this, but also added a few lines describing the usage of handedness by eeprom. Let me know if my changes are good, or if I should trim that section out entirely.